From 16663c9ed6b8bd0c065f28cd34434a4939267bdc Mon Sep 17 00:00:00 2001
From: Michael Schroeder <mls@suse.de>
Date: Thu, 7 Jun 2018 14:18:25 +0200
Subject: [PATCH 3/6] Remove wrong solv_free(data->vincore) in
 repodata_internalize

We're extending the old buffer, so the free is always wrong.

(cherry picked from commit 241566effa2cda6e9445dc596447c43ea4f0719f)
---
 src/repodata.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/repodata.c b/src/repodata.c
index 06b2ea3e..9af4bdf3 100644
--- a/src/repodata.c
+++ b/src/repodata.c
@@ -3512,7 +3512,6 @@ entrydone:
   data->incoredatalen = newincore.len;
   data->incoredatafree = 0;
 
-  solv_free(data->vincore);
   data->vincore = newvincore.buf;
   data->vincorelen = newvincore.len;
 
-- 
2.18.0.rc2

